lib/analyzer/applications/java.ts

```diff
@@ -178,22 +178,21 @@ function unpackJarsTraverse({
       jarPath = `${jarPath}/${entryName}`;
 
       unpackJarsTraverse({
-        jarBuffer: entryData,
+        jarContents: entryData,
         jarPath,
         desiredLevelsOfUnpacking,
         requiredLevelsOfUnpacking,
         unpackedLevels,
         jarBuffers,
         coords,
-        dependencies,
```

Comment:
NOTE: This is the actual fix. Apologies if it gets lost in the rename! I just found that having two variables like `jarBuffer` and `jarBuffers` is super confusing and unclear what relates to what